README.md

```diff
@@ -26,6 +28,10 @@ You can also run with other models, but be warned they may not adhere to the edi
 
     python wolverine.py --model=gpt-3.5-turbo buggy_script.py "subtract" 20 3
 
+If you want to use GPT-3.5 by default instead of GPT-4 add this to `.env`:
+
+    DEFAULT_MODEL=gpt-3.5-turbo
+
```

Comment:
I've just merged another PR that also set up .env usage. Can you fix conflicts and add this DEFAULT_MODEL to the .env.sample?